SALTO-6646: use detailedMessage instead of message in saltoError

shayc331 opened the pull request:
_use detailedMessage instead of message in saltoError_

---

_Additional context for reviewer_

---
_Release Notes_: 
_None_

---
_User Notifications_: 
_None_

dantal4 commented on packages/workspace/src/workspace/workspace.ts:
```diff
@@ -1097,6 +1097,7 @@ export const loadWorkspace = async (
     return {
       ...saltoElemErr,
       message: saltoElemErr.message,
+      detailedMessage: saltoElemErr.detailedMessage,
```
how this didn't break the code in the previous PR?

shayc331 commented:
@dantal4 these changes came after I renamed `message` to `message2`, so I changed every place that used `message` to `detailedMessage`. Am I missing something?

dantal4 reviewed (approved):
💯
